RELEASE-NOTES.txt

```diff
@@ -4,3 +4,5 @@
 - improvement: Present an error message with an option to retry when adding a note to an order fails
 - improvement: Present an error message with an option to retry when fulfilling an order fails
 - bugfix: Log out of the current account right after selecting "Try another account" in store picker
+- improvement: Use the store name for the title of the view in "My store" tab
+- improvement: Add analert to let the user know about our new store switcher
```

Comment:
Barely worth mentioning...

> Add **an alert** to....

😄

Reply:
I'm so proud of @bummytime catching this before I did 👍

Reply:
HAHAHA!

#wordchamp 🥊